tests/backend/checkmach/gen/gen_dune.ml

```diff
@@ -0,0 +1,79 @@
+let () =
+  let enabled_if flambda_only =
+    if flambda_only then
+      (* what we really want to say if dune knew about flambda2:
```

Comment:
(We may promote this comment to
CR/CR-someday to ensure it is revisited.
And maybe ask the dune developers to
add that variable.)

Reply:
Add "CR-soon" to the comment. We use stock dune and while we can ask dune developers to add it, it would make more sense once flambda-backend is at least on opam. The upstream version of the compiler doesn't have flambda2 in its config.

Reply:
Sure, but I think it is not unreasonable to
return a default value when a property is
missing. If memory serves, dune is parsing
the output of `ocamlopt -config` so it is not
an API issue.